packages/back-end/src/models/FeatureModel.ts

```diff
@@ -134,6 +136,36 @@ export async function getAllFeatures(
   );
 }
 
+const _undefinedTypeGuard = (x: string[] | undefined): x is string[] =>
+  typeof x !== "undefined";
+
+export async function getAllFeaturesWithLinkedExperiments(
+  organization: string,
+  project?: string
+): Promise<{
+  features: FeatureInterface[];
+  experiments: ExperimentInterface[];
+}> {
+  const q: FilterQuery<FeatureDocument> = { organization };
+  if (project) {
+    q.project = project;
+  }
+
+  const features = await FeatureModel.find(q);
+  const expIds = new Set<string>(
+    features
+      .map((f) => f.linkedExperiments)
+      .filter(_undefinedTypeGuard)
+      .flat()
+  );
+  const experiments = await getExperimentsByIds(organization, [...expIds]);
```

Comment:
This will run a Mongo query even if the list of ids is empty.  Either call this conditionally or make `getExperimentsByIds` return early if empty.